tests/Api/Responses/Expected/admin/catalog_promotion/post_catalog_promotion_response.json

```diff
@@ -19,6 +19,17 @@
             ]
         }
     ],
+    "actions": [
+        {
+            "@type": "CatalogPromotionAction",
+            "@id": @string@,
```

Comment:
```suggestion
            "@id": "\/api\/v2\/admin\/catalog-promotion-actions\/@integer@",
```
Won't it work?

Reply:
I had a similar observation in PR with rules, but if I'm correct, there is no get endpoint for CatalogPromotionAction, so it doesn't have IRI. We are operating on rules and actions only by operating on CatalogPromotion entity